models/Pokemon.go

```diff
@@ -0,0 +1,319 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/dannegm/academy-go-q12021/constants"
+	"github.com/dannegm/academy-go-q12021/helpers"
+)
+
+// Pokemon represet the basic Pokemon model
+type Pokemon struct {
+	ID           int    `json:"id"`
+	Name         string `json:"name"`
+	TypeOne      string `json:"typeOne"`
+	Total        int    `json:"total"`
+	HealthPoints int    `json:"healthPoints"`
+	Attack       int    `json:"attack"`
+	Defense      int    `json:"defense"`
+	Speed        int    `json:"speed"`
+}
+
+// Pokemon structure for PokeAPI
+
+type PokemonApi struct {
```

Comment:
This is hard to read.

Instead of defining them in place, you could create innermost structs first, and use them as part of the next level structs.

Pro-tip, there's an extension in VS Code that might help you with this, Its name is "Paste JSON as Code", or something like that.